plugins/media-editor/imagecropper/layout/form.php

```diff
@@ -1,62 +1,79 @@
-        <button type="button" class="btn" data-method="scaleY" data-option="-1" title="Flip Vertical"><?php JText::printf('PLG_MEDIA-EDITOR_IMAGECROPPER_FLIP_VERTICAL') ?></button>
+        <button type="button" class="btn" data-method="scaleX" title="<?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_FLIP_HORIZONTAL'); ?>"><?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_FLIP_HORIZONTAL'); ?></button>
+        <button type="button" class="btn" data-method="scaleY" title="<?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_FLIP_VERTICAL'); ?>"><?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_FLIP_VERTICAL'); ?></button>
+    </div>
+
+	<div class="btn-group">
+		<button type="button" class="btn" data-method="zoom" data-option="0.1" title="<?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_ZOOM_IN'); ?>"><?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_ZOOM_IN'); ?></button>
+		<button type="button" class="btn" data-method="zoom" data-option="-0.1" title="<?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_ZOOM_OUT'); ?>"><?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_ZOOM_OUT'); ?></button>
+	</div>
+
+    <div class="btn-group">
+        <button type="button" class="btn btn-primary" data-method="move" data-option="-10" data-second-option="0" title="Move Left">Move Left</button>
+        <button type="button" class="btn btn-primary" data-method="move" data-option="10" data-second-option="0" title="Move Right">Move Right</button>
+        <button type="button" class="btn btn-primary" data-method="move" data-option="0" data-second-option="-10" title="Move Up">Move Up</button>
+        <button type="button" class="btn btn-primary" data-method="move" data-option="0" data-second-option="10" title="Move Down">Move Down</button>
+    </div>
+
+    <div class="btn-group">
+        <button type="button" class="btn btn-primary" data-method="crop" title="Crop">Crop</button>
     </div>
 
 </div>
 
 <div class="cropper-bg">
-    <?php echo JHtml::_('image', COM_MEDIA_BASEURL . '/' . $filePath, '', 'id="media-image"'); ?>
+    <?php echo JHtml::_('image', COM_MEDIA_BASEURL . '/' . $filePath, '', 'id="joomla-media-image-cropper" data-some-attribute="some value" data-url="'. $uploadUrl . '"'); ?>
 </div>
 <br />
-<input type="checkbox" name="save_copy" id="save_copy" checked="checked"/> <label for="save_copy"><?php JText::printf('PLG_MEDIA-EDITOR_IMAGECROPPER_SAVE_COPY') ?></label>
-<input type="submit" name="<?php echo JText::_('PLG_MEDIA-EDITOR_IMAGECROPPER_FIELD_SUBMIT'); ?>" />
+<!-- @TODO: Implement this with javascript, eg parse the old filename and then add a number at the end
+            or something similar, maybe ajax call to see if that filename already exists
+<input type="checkbox" name="save_copy" id="save_copy" checked="checked"/> <label for="save_copy"><?php echo JText::_('PLG_MEDIA_EDITOR_IMAGECROPPER_SAVE_COPY') ?></label>
```

Comment:
I am probably missing the point but why must this be Javascript? What is wrong using PHP for this? We must call the image model to make sure the image is stored in the database as well, not just saved on disk.